src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -261,6 +263,11 @@ private function process_directives_args( string $html, array &$context_stack, a
 			 * We still process the rest of the HTML.
 			 */
 			if ( 'SVG' === $tag_name || 'MATH' === $tag_name ) {
+				if ( $p->get_attribute_names_with_prefix( 'data-wp-' ) ) {
+					/* translators: %s: SVG or MATH HTML tag. */
+					$message = sprintf( __( '%s tags are currently incompatible with the directives processing. Please refrain from adding directives to them.' ), $tag_name );
+					_doing_it_wrong( __METHOD__, $message, '6.6' );
```

Comment:
```suggestion
					_doing_it_wrong( __METHOD__, $message, '6.6.0' );
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6413/commits/415492536fed71329c0280ab94911ea938e6eff6